pex/resolver.py

```diff
@@ -601,7 +602,10 @@ def _check_resolve(self, resolved_distributions):
           )
         else:
           resolved_dist = resolved_requirement_dist.distribution
-          if resolved_dist not in requirement:
+          # If a requirement is resolved vacuously due to a platform constraint (e.g., 'pathlib2; python_version < "3"'
+          # when we're on Python 3) then resolved_dist will be None, and this is not an error.
```

Comment:
I think you might have meant to use a comma:
```suggestion
          # when we're on Python 3, then resolved_dist will be None, and this is not an error.
```

Reply:
No, this closes the paren opened before "e.g.,"